kafka-connector/copy_tool.py

```diff
@@ -17,9 +17,9 @@
 import tempfile
 import subprocess
 
-KAFKA_RELEASE = "2.6.0"
+KAFKA_RELEASE = "3.0.0"
 KAFKA_FOLDER = f"kafka_2.13-{KAFKA_RELEASE}"
-KAFKA_LINK = f"https://downloads.apache.org/kafka/{KAFKA_RELEASE}/{KAFKA_FOLDER}.tgz"
+KAFKA_LINK = f"https://dlcdn.apache.org/kafka/{KAFKA_RELEASE}/{KAFKA_FOLDER}.tgz"
```

Comment:
We can change the link to https://archive.apache.org/dist/kafka/2.6.0/kafka_2.13-2.6.0.tgz instead.

Reply:
Sounds good. We can change it to the archive link instead